src/test/java/com/softserve/edu/greencity/ui/tests/runner/GreenCityTestRunner.java

```diff
@@ -95,6 +94,7 @@ public void tearDownAfterClass() {
 
     @BeforeMethod
     public void setUp() {
+        driver.manage().window().maximize();
```

Comment:
This is important, since some methods don't return the original widow size, and the following tests may fail